GOVERNANCE.MD

```diff
@@ -0,0 +1,7 @@
+# Governance
+
+Project governance as well as policies, procedures and instructions for contributing to FLEDGE can be found on our Wiki site at the following locations:
+
+
+- [FLEDGE Governance Wiki Page](https://wiki.lfedge.org/display/FLEDGE/Governance)
+- [Contributor's Guide](https://github.com/fledge-iot/fledge/blob/develop/CONTRIBUTING.md)
```

Comment:
relative link will be better, as this is within the repository

Reply:
Changed to a relative link.